### ISSUE TITLE
Vertical gaps are completely ignored

### ISSUE THREAD

**Issue by Stooovie** [LoadingByte/cinecred#9]
CC 1.5.1 on MacOS 14.4.

Vertical gaps are completely ignored. No matter if I add a number without units, or with `px`. Gap is set to 32 in Project settings. No gaps are in the actual roll.

![Screenshot 2024-03-24 at 9 55 50](https://github.com/LoadingByte/cinecred/assets/10572373/d05e68db-12bb-42b9-8c8c-98155cbcfaab)
![Screenshot 2024-03-24 at 9 56 02](https://github.com/LoadingByte/cinecred/assets/10572373/51af44c1-e879-452b-93c9-5e0472d3bb09)

**Comment by LoadingByte** [comment]
Is the *Runtime Fine Adjustment* option in the *Project Settings* enabled by any chance? That setting compresses vertical gaps to best achieve the desired runtime. In your case, it looks like it's set to a very low runtime, and thus it completely eliminates all vertical gaps.

**Comment by Stooovie** [comment]
No, it was the Scroll per Frame setting. Maybe add a warning tip even there?

**Comment by LoadingByte** [comment]
*Scroll per Frame* doesn't alter the vertical gaps in any way. It does however change the runtime, and when *Runtime Fine Adjustment* is enabled, that change in runtime leads to a change of the vertical gaps. For example, when increasing *Scroll per Frame*, the runtime decreases, so *Runtime Fine Adjustment* needs to do less compressing to achieve its goal runtime.

I'll think about adding a warning to *Runtime Fine Adjustment* when the gap compression is excessive.

**Comment by Stooovie** [comment]
Thank you! I use CC for TV rolls, and those are usually much faster than film.

**Comment by LoadingByte** [comment]
It took way too long, but Cinecred 1.9.0 just released and finally includes a warning when the gap compression due to *Runtime Fine Adjustment* is excessive.
